examples/basics/visuals/colorbar_visual.py

```diff
@@ -81,7 +83,7 @@ def get_mandlebrot_escape_values(width, height):
     return v_get_num_escape_turns(*grid).astype(np.float)
 
 
-def get_vertical_bar():
+def get_vertical_bar(pos, halfdim):
```

Comment:
I think `halfdim=(x, y)` is not as clear as `size=(major, minor)`, where `major` is the length along the major (typically longer) axis of the colorbar and `minor` is the length along the minor (typically shorter) axis. If we use `size=(major, minor)` everywhere, it's clear what the colorbar should look like.